util/notification/expression/repo/repo.go

```diff
@@ -69,11 +69,8 @@ func FullNameByRepoURL(rawURL string) string {
 	}
 
 	path := gitSuffix.ReplaceAllString(parsed.Path, "")
-	if pathParts := text.SplitRemoveEmpty(path, "/"); len(pathParts) >= 2 {
-		return strings.Join(pathParts[:2], "/")
-	}
-
-	return path
+	pathParts := text.SplitRemoveEmpty(path, "/")
+	return strings.Join(pathParts, "/")
 }
```

Comment:
LGTM. Can you also update the function description in `docs/operator-manual/notifications/functions.md` to make it clear that nested projects are also supported?

Reply:
I updated the documentation. Please let me know if it's ok to you.